uxarray/grid/utils.py

```diff
@@ -320,3 +320,74 @@ def _get_lonlat_rad_face_edge_nodes(
     face_edges_lonlat_rad[valid_mask, 1] = node_lat_rad[valid_edges]
 
     return face_edges_lonlat_rad.reshape(n_face, n_max_face_edges, 2, 2)
+
+
+@njit(cache=True)
+def _edge_nodes_cartesian_kernel(
```

Comment:
Please update their test case in `test_helpers.py`, replacing all calls from `_get_cartesian_face_edge_nodes` to `_edge_nodes_cartesian_kernel`